mii/grpc_related/modelresponse_server.py

```diff
@@ -63,15 +62,15 @@ def _get_model_time(self, model, sum_times=False):
         return model_time
 
     def CreateSession(self, request, context):
-        if request.session_id in self.session_context:
-            raise ValueError(f"session {request.session_id} already exists")
-        self.session_context[request.session_id] = None
+        # TODO improve this so the task is not hard-coded
+        task_methods = GRPC_METHOD_TABLE[Tasks.TEXT_GENERATION]
+        task_methods.create_session(request.session_id)
```

Comment:
@TosinSeg let's figure out how we can verify that the task the user selected in `Task.TEXT_GENERATION` here and error out otherwise. If we don't, we would allow users to do things like select task as `text-classification` and then call `create_session(session_id)` (but this is meaningless for all tasks except `text-generation`.

Reply:
Does my idea in the latest commit look good?